core/src/wallet/ethereum/ERC20/ERC20LikeOperation.h

```diff
@@ -47,7 +47,9 @@ namespace ledger {
                                const Operation &operation,
                                const api::Currency &currency);
 
-            ERC20LikeOperation() {};
+            ERC20LikeOperation() {
+                _blockHeight = 0;
```

Comment:
May you move that into the `ERC20LikeOperation` ctor’s initialization list? Tom Hanks!